src/test/regress/sql/with.sql

```diff
@@ -1046,3 +1046,16 @@ WITH updated AS (
 	update rank_tbl set rank = 6 where id = 5 returning rank
 )
 select count(*) from rank_tbl where rank in (select rank from updated);
+
+-- checking that the planner can build a plan with non-select CTE sharing.
+set gp_cte_sharing to on;
```

Comment:
I think that this line is not needed here, because this parameter is set at the beginning of the file; also, it does not need to be restored at the end.